lib/migrator.js

```diff
@@ -1,90 +1,88 @@
 'use strict';
 
-const FeedParser = require('feedparser');
-const async = require('async');
 const TurndownService = require('turndown');
-const request = require('request');
-const url = require('url');
-const fs = require('fs');
-
-exports.registerMigrator = function(hexo) {
-  hexo.extend.migrator.register('rss', (args, callback) => {
-    const source = args._.shift();
-    const tomd = new TurndownService();
-
+const request = require('request-promise-native');
```

Comment:
Maybe `got` or `node-fetch` could be considered?

Reply:
I didn't realize request is under [maintenance mode](https://github.com/request/request/issues/3142) (only knew it through [`got`](https://www.npmjs.com/package/got)). `got` looks promising, I'll have a look.

Reply:
Refactored to `got`.